webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -139,18 +135,17 @@ final case class ProjectsResponderADM(actorDeps: ActorDeps, cacheServiceSettings
    * Given a set of project IRIs, gets the ontologies that belong to each project.
    *
    * @param projectIris    a set of project IRIs. If empty, returns the ontologies for all projects.
-   * @param requestingUser the requesting user.
    * @return a map of project IRIs to sequences of ontology IRIs.
    */
-  private def getOntologiesForProjects(projectIris: Set[IRI], requestingUser: UserADM): Future[Map[IRI, Seq[IRI]]] = {
+  private def getOntologiesForProjects(projectIris: Set[IRI]): Future[Map[IRI, Seq[IRI]]] = {
     def getIriPair(ontology: OntologyMetadataV2) =
       ontology.projectIri.fold(
         throw InconsistentRepositoryDataException(s"Ontology ${ontology.ontologyIri} has no project")
       )(project => (project.toString, ontology.ontologyIri.toString))
 
     val request = OntologyMetadataGetByProjectRequestV2(
       projectIris = projectIris.map(_.toSmartIri),
-      requestingUser = requestingUser
+      requestingUser = KnoraSystemInstances.Users.SystemUser
```

Comment:
I am not entirely sure if it is safe to replace the user here with the the `SystemUser` as this gives the most rights to everyone, shouldn't it be the `AnonymousUser` here?

Reply:
I do not know if  is safe to remove the `requestingUser` at all. It possibly changes the behaviour of the endpoint for logged in users. 

The `getOntologiesForProjects` is called with the `SystemUser` from `getProjectFromTriplestore`. But with the `AnonymousUser` from the `projectsGetADM` in case a user is not logged in.

Reply:
It feels a bit risky to do this... but given the fact that requesting projects doesn't change depending on authentication, and neither does getting ontology metadata, this should be safe. (And if it's not, then there is a major flaw in our system.)

Reply:
I am ok with that change, Irina and I have looked into it and it seems fine.

Reply:
As discussed, I double-checked that the requesting user is not checked/used in the end and hence it is safe to use the `SystemUser` instead of a `requestingUser` in this location.